lang/en/plagiarism_copyleaks.php

```diff
@@ -105,6 +106,9 @@
 $string['clpendingduplication'] = 'Pending Duplication';
 $string['clfailedduplication'] = 'Duplication Failed';
 
+$string['clreportupdatefailed'] = 'Failed to update the report';
+$string['clsendnotificationfailed'] = 'Failed to send notification';
+
 $string['privacy:metadata:core_files'] = 'Copyleaks stores files that have been uploaded to Moodle to form a Copyleaks submission.';
 $string['privacy:metadata:plagiarism_copyleaks_files'] = 'Information that links a Moodle submission to a Copyleaks submission.';
 $string['privacy:metadata:plagiarism_copyleaks_files:userid'] = 'The ID of the user who is the owner of the submission.';
```

Comment:
@ShadeAma all translations that were added to this file - need to be added to the "fr" file too